consensus/spos/worker.go

```diff
@@ -463,6 +470,13 @@ func (wrk *Worker) ProcessReceivedMessage(message p2p.MessageP2P, fromConnectedP
 		wrk.doJobOnMessageWithSignature(cnsMsg, message)
 	}
 
+	if isMessageWithInvalidSigners {
```

Comment:
no need for a activation epoch here? If a message with invalid signers comes before the activation, the old handling of "ProcessReceivedMessage" will be different.

Reply:
added early exit, indeed it may happen

Reply:
is it added somewhere else?

Reply:
removed it, forgot to update the comment

as discussed, even though a message would be received before activation, the cache should not have any data, thus this verification should always return nil

Reply:
ok